packages/daf-express/src/did-doc-router.ts

```diff
@@ -0,0 +1,89 @@
+import { IAgent, IIdentity, IIdentityManager, TAgent } from 'daf-core'
+import { Request, Response, NextFunction, Router, json } from 'express'
+import { getOpenApiSchema } from 'daf-rest'
+import Debug from 'debug'
+
+interface RequestWithAgentIdentityManager extends Request {
+  agent?: TAgent<IIdentityManager>
+}
+
+interface RequestWithAgent extends Request {
+  agent?: IAgent
+}
+
+export const didDocEndpoint = '/.well-known/did.json'
+/**
+ * @public
+ */
+export interface DidDocRouterOptions {
+  /**
+   * Function that returns configured agent for specific request
+   */
+  getAgentForRequest: (req: Request) => Promise<TAgent<IIdentityManager>>
+}
+/**
+ * Creates a router that serves `did:web` DID Documents
+ *
+ * @param options - Initialization option
+ * @returns Expressjs router
+ */
+export const DidDocRouter = (options: DidDocRouterOptions): Router => {
```

Comment:
Perhaps we should be more explicit about serving a `did:web` here by naming this `WebDidDocRouter`

Reply:
Makes sense. Renamed to `WebDidDocRouter`